packages/aws-cdk/lib/init-templates/app/csharp/src/%name.PascalCased%/Program.template.cs

```diff
@@ -10,7 +10,7 @@ sealed class Program
         public static void Main(string[] args)
         {
             var app = new App();
-            new %name.PascalCased%Stack(app, "%stackname%", new StackProps
+            new %PascalStackName%(app, "%stackname%", new %PascalStackProps%
```

Comment:
This shouldn't change `StackProps`

Reply:
Line 6 of this template is what we discussed changing, not line 13.

Reply:
if the stack is expecting our own stack props then we can't use StackProps or it would give us

```
error CS1503: Argument 3: cannot convert from 'Amazon.CDK.StackProps' to 'GoodCSharp.GoodCSharpStackProps' [/Users/bobertzh/work/deleteme/temp/GoodCSharp/src/GoodCSharp/GoodCSharp.csproj]

```

Reply:
I did realize though that we actually need more changes on 13 and on 6 since we aren't going to differentiate between stack name and app name. I'll push a change